src/components/Book.js

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import { useSelector, useDispatch } from 'react-redux';
+import { removeBookAsync } from './redux/books/bookSlice';
+
+const Book = () => {
+  const books = useSelector((state) => state.book.books);
+  const dispatch = useDispatch();
+
+  const handleRemoveBook = (bookId) => {
+    dispatch(removeBookAsync(bookId));
+  };
+
+  return (
+    <div className="book-container">
+      {books.map((book) => (
+        <div key={book.id}>
+          <h3>{book.title}</h3>
+          <p>{book.author}</p>
+          <span>
+            <button type="button" onClick={() => handleRemoveBook(book.id)}>
+              Remove
+            </button>
+          </span>
+        </div>
+      ))}
+    </div>
+  );
+};
```

Comment:
- [ ] As required for the project, your application to still work as expected, kindly ensure that user can remove a book. Right now, book can be added but it cannot be removed. Also the `Check Status` button is not working as expected, it doesn't render the expected message

Reply:
Hi@tresorsawasawa 
I appreciate your feedback and will address it.